debian/build.go

```diff
@@ -67,11 +67,10 @@ func (ss *stringList) Set(v string) error {
 }
 
 var (
-	architectures = stringList{"amd64", "arm", "arm64", "ppc64le", "s390x"}
-	serverDistros = stringList{"bionic"}
-	allDistros    = stringList{"bionic", "xenial", "jessie", "precise", "sid", "stretch", "trusty", "utopic", "vivid", "wheezy", "wily", "yakkety"}
-	kubeVersion   = ""
-	revision      = "00"
+	architectures           = stringList{"amd64", "arm", "arm64", "ppc64le", "s390x"}
+	distros                 = stringList{"bionic", "xenial", "trusty", "stretch", "jessie", "sid"}
```

Comment:
Can we add [Debian Buster](https://wiki.debian.org/DebianBuster), which is due to be released soon™?

Reply:
@praseodym -- Buster is scheduled to be released on the same day as 1.15. We can add it for 1.16 though. Can you open an issue in this repo for that?

Reply:
@praseodym -- I've opened a tracking issue for this: https://github.com/kubernetes/release/issues/728